deployment/src/main/java/io/quarkiverse/cxf/deployment/QuarkusCxfProcessor.java

```diff
@@ -277,6 +277,14 @@ public void build(
                     String impl = wsClass.name().toString();
                     String wsName = impl.contains(".") ? impl.substring(impl.lastIndexOf('.') + 1) : impl;
                     additionalBeans.produce(AdditionalBeanBuildItem.unremovableOf(impl));
+
+                    /*
+                     * Address issue #346; though this really should be handled in ArC
+                     * https://github.com/quarkiverse/quarkus-cxf/issues/346
+                     * https://github.com/quarkusio/quarkus/issues/7507
+                     */
+                    reflectiveClass.produce(new ReflectiveClassBuildItem(true, true, impl + "_Subclass"));
```

Comment:
Do we really need fields or are methods enough?

Reply:
I thought we may need both from looking at `SubclassGenerator` - but now that you asked, I tested it and actually seems we just need methods.  Fixed in latest commit.